addons/l10n_tr_nilvera/models/res_partner.py

```diff
@@ -48,6 +48,16 @@ def _compute_nilvera_customer_alias_id(self):
         for record in self:
             record.l10n_tr_nilvera_customer_alias_id = record.l10n_tr_nilvera_customer_alias_ids[:1]
 
+    def _compute_is_company(self):
+        l10n_tr_partners = self.filtered(lambda p: p.vat and p.country_code == 'TR')
```

Comment:
Isn't the check on country code sufficient, as the vat check is done below ? Allows to be sure other overrides / base method won't have side effects.